tests/puppeteer/pages/BO/advancedParameters/email/index.js

```diff
@@ -0,0 +1,67 @@
+require('module-alias/register');
+const BOBasePage = require('@pages/BO/BObasePage');
+
+module.exports = class Emails extends BOBasePage {
```

Comment:
Is it `Emails` or `Email` ? Do we use singular ?

Reply:
Yes Email :) I will fix it 
Thank you